src/mono/wasm/runtime-test.js

```diff
@@ -7,42 +7,65 @@
 var is_browser = typeof window != "undefined";
 var consoleWebSocket;
 var print;
+var printErr;
+var make_print_fn;
 
 if (is_browser) {
-	// We expect to be run by tests/runtime/run.js which passes in the arguments using http parameters
+	make_print_fn = function(level) {
+		return function(_msg) {
+			window.real_print(JSON.stringify({
+				method: `console.${level}`,
+				payload: _msg
```

Comment:
Thoughts on passing all the args?
```suggestion
				payload: {...arguments}
```

Reply:
If there is more than one argument, then how should that be logged? as the json array itself, so `[ "arg1", 5, {x: 4} ]`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Console/debug

Reply:
preserve as much fidelity as you can?

Reply:
Basically we treat console as an object so something like
```
realConsole[method](...payload);
```

Reply:
consider:

```javascript
var message = "time";
var domain = "fun";
var fmt = "%s %s";
console.log (fmt, domain, message);
```
the output should be 

```
fun time
```

Reply:
so
```javascript
var payload = ["%s %s", "fun", "time"];
var method = "log";
console[method](...payload);
```
outputs 
```
fun time
```